app/services/sign_in/constants/service_account_access_token.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module SignIn
+  module Constants
+    module ServiceAccountAccessToken
+      VALIDITY_LENGTH_SHORT_MINUTES = 5.minutes
+      JWT_ENCODE_ALGORITHM = 'RS256'
+
+      VERSION_LIST = [
+        CURRENT_VERSION = 'V0'
+      ].freeze
+
+      ISSUER = '/v0/sign_in'
```

Comment:
Should the issuer be the path? The is going to change when we migrate. Could the issuer be something generic like `va_sign_in_service`?